common/gce.py

```diff
@@ -88,3 +88,56 @@ def get_base_target_link(experiment_config):
 def get_instance_from_preempted_operation(operation, base_target_link) -> str:
     """Returns the instance name from a preempted |operation|."""
     return operation['targetLink'][len(base_target_link):]
+
+
+def get_instance_group_managers():
+    """Returns the instance group managers resource."""
+    return thread_local.service.instanceGroupManagers()
+
+
+def get_instance_group_size(instance_group: str, project: str,
```

Comment:
Can you create an InstanceGroupManager class and keep project, zone, instance_group in __init__, it is repeated across all functions. functions become self.size, self.resize, self.delete, self.create

Reply:
Need to think about how I want to do this. I think I will land as is for now.